### PR TITLE
ER-972 provider navigation menu

### PR DESCRIPTION
**Pull request by sachpatel**
This is basic.

Going forward we will be using a shared partial across DAS for the employer and Provider menu's these should also take not of the whitelisting and env differences for showing particular menu items.

I have copied in the way that the footer is on the PAS website on test, including a survey link to AccessMyLevy, we can change this as we get a survey for provider recruit or drop it all together.

I have left out the reservations link which might appear on the other env's because it is not showing up on TEST yet. We will have to revisit this when going through the env's to make sure the menu items are the same across other das services.

### CODE REVIEW

File: src/Provider/Provider.Web/Controllers/ExternalLinksController.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	public class ExternalLinksController : Controller
+    {
+        private readonly ProviderApprenticeshipsLinkHelper _linkHelper;
+
+        public ExternalLinksController(ProviderApprenticeshipsLinkHelper linkHelper)
+        {
+            _linkHelper = linkHelper;
+        }
+
+        [HttpGet(RoutePaths.AccountRoutePath + "/account-home", Name = RouteNames.Dashboard_Account_Home)]
```
Comment (chrisjensenuk) [inline src/Provider/Provider.Web/Controllers/ExternalLinksController.cs]: Can we put `RoutePaths.AccountRoutePath` as a class level attribute?

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionBase.cs [hunk 2]
```diff
@@ -39,8 +39,9 @@ protected IMongoCollection<T> GetCollection<T>()
         {
             var settings = MongoClientSettings.FromUrl(new MongoUrl(_config.ConnectionString));
             settings.SslSettings = new SslSettings { EnabledSslProtocols = SslProtocols.Tls12 };
+            settings.ConnectionMode = ConnectionMode.Direct;
```
Comment (chrisjensenuk) [inline src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionBase.cs]: What does this do?

File: src/Provider/Provider.Web/Controllers/ExternalLinksController.cs (hunk 1 above)
Comment (sachpatel) [reply src/Provider/Provider.Web/Controllers/ExternalLinksController.cs]: 👍

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionBase.cs (hunk 2 above)
Comment (sachpatel) [reply src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionBase.cs]: 😱 mistake, will remove.

**Pull request merged**